src/emailpro/account/emailpro-account.controller.js

```diff
@@ -1,20 +1,26 @@
-angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtrl', ($scope, EmailPro, $q, $stateParams, $translate) => {
+import {
+  SIZE_UNIT, STATE_TASK_DOING, STATE_TASK_ERROR, ACCOUNT_TYPES, FILTER_TYPE,
+} from './emailpro-constants';
+
+angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtrl', ($scope, EmailPro, $q, $stateParams, $translate, WucConverterService) => {
   $scope.stateCreating = EmailPro.stateCreating;
   $scope.stateDeleting = EmailPro.stateDeleting;
   $scope.stateOk = EmailPro.stateOk;
   $scope.stateReopening = EmailPro.stateReopening;
   $scope.stateSuspended = EmailPro.stateSuspended;
   $scope.stateSuspending = EmailPro.stateSuspending;
+  $scope.WucConverterService = WucConverterService;
```

Comment:
Injecting this service directly to the `$scope` doesn't make sense because you only refer to that service in this controller.

Could you please remove it?

Thanks!